### DISCUSSION TITLE
Why `Idle Parse Delay` is so long by default (2 sec)

### DISCUSSION THREAD

**Discussion by me2beats** [godotengine/godot-proposals#4625] [Editor]
Usually I set it to 0.5
that makes the script editor more responsive
![изображение](https://user-images.githubusercontent.com/16458555/172031944-8e65dc33-b0fb-4b47-9209-ad3a133530cb.png)

**Comment by Calinou** [comment]
See discussion in https://github.com/godotengine/godot/pull/25751.

In 4.0, GDScript is now able to report multiple errors at once (and has working autocompletion if there's an error). That said, some people don't like seeing errors while they're working on their code.

I think we could try decreasing this value to 1.5 by default, as this seems closer to what most other code editors are using.

**Edit:** Pull request opened: https://github.com/godotengine/godot/pull/61814

**Reply by me2beats** [reply]
well yes, I know some users who are annoyed by the frequent errors. But for some reason I got used to it, it helps me to check the code faster. And this is even considering that I don't type fast.

and now when I download a new version or run the editor in self-contained mode, I immediately feel this difference.
